### ISSUE TITLE
Startup Crash Error

### ISSUE THREAD

**Issue by alpha1178** [e621-ReBot/e621-ReBot-v2#1]
I get this .NET Framework error upon startup.

************** Exception Text **************
System.IO.FileLoadException: Could not load file or assembly 'CefSharp.WinForms, Version=103.0.120.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
File name: 'CefSharp.WinForms, Version=103.0.120.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138'
   at e621_ReBot_v2.Modules.Module_CefSharp.InitializeBrowser(String WebAdress)
   at e621_ReBot_v2.Form_Loader.Form_Loader_Load(Object sender, EventArgs e) in E:\VS Projects\e621 ReBot v2\e621 ReBot v2\Forms\Form_Loader.cs:line 42
   at System.Windows.Forms.Form.OnLoad(EventArgs e)
   at System.Windows.Forms.Form.OnCreateControl()
   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   at System.Windows.Forms.Control.CreateControl()
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Form.WmShowWindow(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

**Comment by alpha1178** [comment]
Version is the latest: V 2.2.5.0 trying to download 2.2.6.0

**Comment by e621-ReBot** [comment]
Can you explain how you got it? You extracted the 2.2.5.0 release and just tried to launch it, or you had earlier version and tried to update?
If you updated, I suggest you simply delete the folder and extract it again (and if you need the settings, then go to appdata using the button in the app and just copy the settings from old to new folder and relaunch it).

**Comment by alpha1178** [comment]
Started with just launching it. Got the error, selected quit. Tried to start it again and this time I opted to continue which just went to update the program but still doesn’t show up. In task manager, it’s there but it’s not running exactly. Left it for 30 minutes and still nothing. Closed the task and continued to try again to no avail. 

I tried following the steps from the discord but nothing worked. Yes, I did overwrite the files from the zip to the folder where the program is located.
